gcore/src/general.rs

```diff
@@ -16,8 +16,8 @@
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
-#[derive(Clone, Copy, Debug, PartialEq, Eq)]
-pub struct MessageHandle(pub(crate) u32);
```

Comment:
Continue to use `pub(crate)` is still a rational idea, as it seems to me.

Reply:
I don't think giving access to internal field of `MessageHandle` is makes sense outside of `msg` module. If could be added later if that proves to be necessary.